src/graphnet/models/detector/icecube.py

```diff
@@ -43,6 +43,34 @@ def _forward(self, data: Data) -> Data:
         return data
 
 
+class IceCube86Kaggle(Detector):
```

Comment:
Does the Kaggle data represent the main array only or also DeepCore?

Reply:
Both!

Reply:
Then I figure it should be named `IceCubeDeepCoreKaggle` or just `IceCubeKaggle`.

Reply:
It has been renamed to `IceCubeKaggle`